cunumeric/random/generator.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+from cunumeric.random.bitgenerator import XORWOW, BitGenerator
+
+
+class Generator:
```

Comment:
any reason you left out `uniform`? I feel it's straightforward to add it given that you already have the implementation.

Reply:
missing test and last API link - fixed in commit 7364d28